packages/@lwc/synthetic-shadow/src/faux-shadow/html-element.ts

```diff
@@ -159,6 +171,28 @@ defineProperties(HTMLElement.prototype, {
         writable: true,
         configurable: true,
     },
+    innerText: {
+        get(this: Element): string {
+            if (!featureFlags.ENABLE_ELEMENT_PATCH) {
+                if (isNodeShadowed(this) || isHostElement(this)) {
+                    return getInnerText.call(this);
+                }
+
+                return innerTextGetter.call(this);
+            }
+
+            // TODO [#1222]: remove global bypass
+            if (isGlobalPatchingSkipped(this)) {
+                return innerTextGetter.call(this);
+            }
+            return getInnerText.call(this);
+        },
+        set(v: string) {
+            innerTextSetter.call(this, v);
```

Comment:
This might not be sufficient 🤔 , add tests to make sure it works as expected.

Reply:
yeah, i haven't started with that, from the pr desc:  _Pending: innerText setter implementation._